corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}{% load compress %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Doc {{ doc_id }}</title>
+        {% compress css %}
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'style/less/hqstyle-core.less' %}"/>
+        {% endcompress %}
+        {% if less_debug %}
+            <script src="{% static 'style/lib/lessjs/less-1.3.1.min.js' %}"></script>
+            <script>less.watch();</script>
+        {% endif %}
+    </head>
+    <body>
+        <div class="container-fluid">
+            <div class="page-header">
+                <h1>Enter a doc id</h1>
+                <form class="form-horizontal" method="GET" action="">
+                    <input name="id" type="text" style="width: 30em"/>
```

Comment:
This may be possible, but it looks like it'd be something along the lines of:
```html
<div class="form-group">
    <div class="input-group">
        <div class="set-width-to-30-em">
            <input name="id" class="form-control" type="text"/>
        </div>
    </div>
</div>
```
Gonna punt